src/specs/utils.spec.js

```diff
@@ -0,0 +1,65 @@
+import moment from 'moment';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
import moment from "moment";
```